test/test-load.ts

```diff
@@ -1,53 +1,58 @@
 import * as io from '../lib/io';
 import * as mqtt from '../lib/mqtt';
 import { Md5Hash, hash_md5 } from '../lib/crypto';
+import ResourceSafety = require('../lib/resource_safety');
+import { TextDecoder } from 'util';
 
 console.log('ALPN is available: ', io.is_alpn_available());
 
 const test_topic = "test";
 
 async function main() {
     let bootstrap = new io.ClientBootstrap();
-    let tls_opt = io.TlsContextOptions.create_client_with_mtls("iot-certificate.pem.crt", "iot-private.pem.key")
-    tls_opt.override_default_trust_store(undefined, "AmazonRootCA1.pem")
-    tls_opt.alpn_list = "x-amzn-mqtt-ca"
-    let tls_ctx = new io.ClientTlsContext(tls_opt);
-    let client = new mqtt.Client(bootstrap, tls_ctx);
+    let config_builder = 
+    mqtt.AwsIotMqttConnectionConfigBuilder.new_mtls_builder_from_path("/home/ANT.AMAZON.COM/henso/source/4d169378e4.cert.pem", 
```

Comment:
Should make these arguments, no one else has your home directory :)

Reply:
well, it's what it was before (something on colden's desktop). Yes, I should go back and do that.